SoftFox/SoftFox/InitialisationError.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+//forward declaration of class
+class InitialisationError
+{
+//make the class declaration public so other cpp files can access it
+public:
+	//the class requires a field input of msg with type string
+	InitialisationError(const std::string& msg);
+	const char* what() { return message.c_str(); }
+
+//What does this do?
```

Comment:
I have read through and dont understand it either we can go through it together in the next pair programming session.